### ISSUE TITLE
There is a big gap between the accuracy of Miou running in mmseg and that in the paper

### ISSUE THREAD

**Issue by tthupup** [DingXiaoH/RepLKNet-pytorch#35]
There is a big gap between the accuracy of Miou running in mmseg and that in the paper, and the running configuration is replknet-31b_ 22Kpretrain_ upernet_ 160k_ ade20k_ 640.py. The environment is a 3080ti GPU of jupyter. Do you know the reason

**Comment by DingXiaoH** [comment]
Hi, I would appreciate it if you could provide your command and customized config file. Our config file was used with 4x8 or 8x8 GPUs. If you have only one GPU, you need to train with more iterations to ensure the same number of training samples.

**Comment by tthupup** [comment]
Thank you for your reply! I only have a 3080ti GPU. Is it enough to adjust the number of iterations from 40000 to 160000 under the same parameters, or do you need more iterations?
